src/UI/Factory.php

```diff
@@ -294,8 +294,9 @@ public function button();
      *           The focus MAY leave the Dropdown if tab is pressed while focusing the last
      *           element. This differs from the behaviour in Popovers and Modals.
      *      8: >
-     *           Dropdowns MUST set the aria-label describing the contained options, if the
-     *           term "Actions" does not describe the contained options well.
+     *           If the description of the contained options are not already given by the component containing the
+     *           dropdown or the Button triggering it, then it MUST be set with the aria-label.
+     *           If the aria-label is just a the Buttons title, it MUST be omitted.
```

Comment:
Thank you very much! There is a little grammatical error in the last sentence, I assume you meant:
```suggestion
     *           If the aria-label is just the title of the corresponding Button, it MUST be omitted.
```

Reply:
Just another tiny comment: "button" in this context is used as a general term and therefore written in lower-case letters. If it would be a proper name (like "Repository" in our UI), using a uppercase "B" would be fine. 

P.S.: I have to confess that our English language file needs a streamlining in the use of capital letters... It's on my list...